Resources/Prototypes/_Scp/Entities/Mobs/NPCs/silicon.yml

```
@@ -0,0 +1,12 @@
+- type: entity
+  parent: MobCleanBot
+  id: MobRoomba
+  name: roomba
+  description: The creep of automation now threatening janitors.
+  components:
+  - type: Sprite
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 3

(indentation)